src/app/screens/Accounts/Detail/index.tsx

```diff
@@ -148,20 +148,23 @@ function AccountScreen() {
         throw new Error("No account available");
       }
 
-      // Validate the private key before saving
-      nostrPrivateKey && generatePublicKey(nostrPrivateKey);
-      nostrPrivateKey && nostrlib.hexToNip19(nostrPrivateKey, "nsec");
-
-      await msg.request("nostr/setPrivateKey", {
-        id: account.id,
-        privateKey: nostrPrivateKey,
-      });
-
       if (nostrPrivateKey) {
+        // Validate the private key before saving
+        nostrPrivateKey && generatePublicKey(nostrPrivateKey);
```

Comment:
Why `nostrPrivateKey &&`?

Reply:
I guess because `generatePublicKey` is expecting a value

Reply:
Oops, I forgot to remove that

> I guess because generatePublicKey is expecting a value

I used to do this without the if statement, so I copy-pasted it :P